### ISSUE TITLE
p-s v2.4.0 fails to install

### ISSUE THREAD

**Issue by boneskull** [kentcdodds/p-s#49]
looks like `dist/` didn't get published, if I had to wager a guess:

```
> $ npm install p-s@latest                                                                                   ⬡ 6.4.0
npm WARN enoent ENOENT: no such file or directory, open '/private/tmp/package.json'
npm WARN tmp No description
npm WARN tmp No repository field.
npm WARN tmp No README data
npm WARN tmp No license field.
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/boneskull/.nvm/versions/node/v6.4.0/bin/node" "/Users/boneskull/.nvm/versions/node/v6.4.0/bin/npm" "install" "p-s@latest"
npm ERR! node v6.4.0
npm ERR! npm  v3.10.3
npm ERR! path /private/tmp/node_modules/p-s/dist/bin/p-s.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/private/tmp/node_modules/p-s/dist/bin/p-s.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/private/tmp/node_modules/p-s/dist/bin/p-s.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /private/tmp/npm-debug.log
```

**Comment by boneskull** [comment]
(I'd send a PR to fix this, but I'm unfamiliar with `semantic-release`)

**Comment by kentcdodds** [comment]
Let me know if that didn't fix it in 2.4.1.
Sorry about that. I messed something up and had to release locally which I don't do often and I guess I forgot to run the build!
